src/Rector/MethodCall/WebTestCaseAssertResponseCodeRector.php

```diff
@@ -148,10 +152,14 @@ private function processAssertResponseStatusCodeSame(MethodCall $methodCall): ?M
             return null;
         }
 
+        if ($methodCall instanceof StaticCall) {
+            return $this->nodeFactory->createStaticCall('self', 'assertResponseStatusCodeSame', [$methodCall->args[0]]);
+        }
+
         return $this->nodeFactory->createLocalMethodCall('assertResponseStatusCodeSame', [$methodCall->args[0]]);
     }
 
-    private function processAssertResponseRedirects(MethodCall $methodCall): ?MethodCall
+    private function processAssertResponseRedirects(Expr\CallLike $methodCall): ?Expr\CallLike
```

Comment:
using union for exact types should can be applied:

```suggestion
    private function processAssertResponseRedirects(MethodCall|StaticCall $methodCall): MethodCall|StaticCall|null
```

Reply:
For some reason I thought I couldn't use this for backwards compatibility, but I just never checked.